.github/ISSUE_TEMPLATE/__en-US.yaml

```diff
@@ -1,6 +1,10 @@
 name: English
 description: Report an issue
 body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to file an issue. It may take awhile for the community to respond. Please visit [Triaging Minikube Issues](https://minikube.sigs.k8s.io/docs/contrib/triage/) if you would like to contribute.
```

Comment:
lets make sure this is pointed out as footnote and it is added as a comment (so it wont show up in the actual markdown)

Reply:
Yes, implemented suggestion.

Reference:
https://stackoverflow.com/questions/4823468/comments-in-markdown